Library/Homebrew/attestation.rb

```diff
@@ -62,9 +62,6 @@ def self.enabled?
       return false if ENV.fetch("CI", false)
       return false if OS.unsupported_configuration?
 
-      gh_version = Formula["gh"].any_installed_version
-      return false if gh_version.nil? || gh_version < "2.49"
```

Comment:
Ok with merging this but the fix will be:

```rb
return false if gh_version.nil? || gh_version.version < "2.49"
```

Reply:
Yeah, I'll do a follow-up with a testcase.